generator/03-syndesis-verifier.yml.mustache

```diff
@@ -94,12 +94,6 @@
               port: 8080
               scheme: HTTP
             initialDelaySeconds: 10
-          livenessProbe:
-            httpGet:
-              path: /health
-              port: 8080
-              scheme: HTTP
-            initialDelaySeconds: 180
```

Comment:
I'm just about to prepare a PR where instead of removing the liveness probe, adding a longer timeout than the default 1s. That worked for @dsimansk , so would like to try it here, too, as I think livenessProbes are important (and in fact would like to add one for syndesis-rest, too)